10_Examples/linh-onboarding/test/test.cpp

```diff
@@ -0,0 +1,37 @@
+#include <gtest/gtest.h>
+#include "linh-onboarding/subscriber.hpp"
+#include <iostream>
+
+
+class TestExample : public ::testing::Test {
+protected:
+
+	TestExample(){
+		subscriber_node_ = std::make_shared<onboarding::Subscriber>();
+	}
+
+	void SetUp() override {
+	}
+
+	void TearDown() override {
+	}
+	std::shared_ptr<onboarding::Subscriber> subscriber_node_;
+};
+
+TEST_F(TestExample, testAdd){
+	EXPECT_EQ(subscriber_node_->add(2, 3), 5);
+	EXPECT_TRUE(subscriber_node_->add(2, 3) == 5);
```

Comment:
these two tests are doing the exact same thing, so dont need to have both of them